roofit/roofitcore/inc/RooFuncWrapper.h

```diff
@@ -41,12 +44,24 @@ class RooFuncWrapper final : public RooAbsReal {
 protected:
    double evaluate() const override;
 
+   void buildCode(RooAbsReal &head, std::string &code, RooArgSet const & /* paramSet */, RooArgSet const &ObsSet,
+                  const RooAbsData *data);
+
 private:
    void updateGradientVarBuffer() const;
 
    using Func = double (*)(double *, double *);
    using Grad = void (*)(double *, double *, double *);
 
+   double doEval(const double *x) const;
```

Comment:
```suggestion
```
The `doEval` function is unused, now that the interface to the minimizer is handled inside `RooMinimizer` you can remove it (also in the `cxx` file).